support-frontend/assets/components/subscriptionCheckouts/orderSummary/orderSummary.jsx

```diff
@@ -33,18 +41,22 @@ function OrderSummary(props: PropTypes) {
         <div css={styles.textBlock}>
           <h3>{props.title}</h3>
           <p>{priceString}</p>
-          <span>14 day free trial</span>
+          {props.productType === DigitalPack && <span>14 day free trial</span>}
```

Comment:
All of these `productType === x` conditionals make me wonder if it would be better to have separate components for the different products. What do you think?

Reply:
Yes, I went back and forth on this quite a bit. I felt it was basically okay for the order summary to be shared and the end summary to be separate, but I could be swayed!

Reply:
I'll leave it up to you to decide, I generally try to avoid if statements as much as possible, particularly when you're having to do it multiple times in the same component

Reply:
Okay - I have separated them. I agree that the components are easier to read now.